runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -149,6 +151,8 @@ public void clientMainLoop()
 			client.setTickCount(tick + 1);
 		}
 
+		eventBus.post(BEFORE_RENDER);
```

Comment:
Maybe this should be a 'main loop' event? Also should possibly be in a separate commit

Reply:
I feel like 'main loop` could be confused with a client tick event